pr_agent/agent/pr_agent.py

```diff
@@ -70,6 +73,8 @@ async def handle_request(self, pr_url, request, notify=None) -> bool:
             if notify:
                 notify()
             await PRReviewer(pr_url, is_answer=True, args=args).run()
+        elif action == "auto_review":
+            await PRReviewer(pr_url, is_auto=True, args=args).run()
```

Comment:
**Suggestion:** Consider adding logging for the 'auto_review' action in the 'handle_request' method. This will help in debugging and understanding the flow of the program.
```suggestion
        elif action == "auto_review":
            logging.info(f'Running auto review for PR: {pr_url}')
            await PRReviewer(pr_url, is_auto=True, args=args).run()
```